src/crypto/CHIPCryptoPAL.h

```diff
@@ -492,6 +492,20 @@ class P256Keypair : public P256KeypairBase
      **/
     CHIP_ERROR Initialize(ECPKeyTarget key_target) override;
 
+    /**
+     * @brief Import a raw key pair (bytes) into the keypair
+     * @return Returns a CHIP_ERROR on error, CHIP_NO_ERROR otherwise
+     **/
+    CHIP_ERROR ImportRawKeypair(const uint8_t * key_data, size_t key_data_size);
+    CHIP_ERROR ImportRawKeypair(ByteSpan private_key, ByteSpan public_key)
+    {
+        return ImportRawKeypair(private_key.data(), private_key.size(), public_key.data(), public_key.size());
+    }
+
+    CHIP_ERROR ImportRawKeypair(const uint8_t * private_key, const size_t private_key_size, const uint8_t * public_key,
```

Comment:
How about we just keep variants that take `ByteSpan`s to keep the interface simple? It's easy to convert pointer + size to a byte span if one needs that.